packages/gradle/src/utils/exec-gradle.ts

```diff
@@ -31,12 +31,9 @@ export function getGradleBinaryPath() {
 
 export function execGradleAsync(
   args: ReadonlyArray<string>,
-  execOptions: ExecFileOptions
+  execOptions?: ExecFileOptions
 ) {
   const gradleBinaryPath = getGradleBinaryPath();
-  if (!existsSync(gradleBinaryPath)) {
-    throw new Error('Gradle is not setup. Run "gradle init"');
-  }
```

Comment:
I don't think there was anything wrong this. Can you bring it back please?

Reply:
this check is a part of getGradleBinaryPath function, which got called before this line. i don't think we need this